### PR TITLE
Grid refactor

### PR DESCRIPTION
**Pull request by webinista**
Suggested fix for #971

### CODE REVIEW

**Comment by msecret** [comment]
This technically looks good to me. We might want to test on a few sites that have implemented the standards just in case the change in specificity broke some custom CSS code, although I'm not sure that's even necessary (especially considering that we haven't come up with that sort of testing structure yet).

**Comment by maya** [comment]
@webinista we did some major restructuring to the repo, can you rebase this off of `18f-pages-staging` to resolve the merge conflict? 

`git fetch --all && git rebase origin/18f-pages-staging`

Our team can also open a new PR to do this if you prefer.

**Comment by webinista** [comment]
@maya okay ... turns out that I had to add the original repo as another remote, and run `git fetch --all && git rebase upstream/18f-pages-staging`. I've (finally) resolved that merge conflict and I think I did it correctly.

File: src/stylesheets/core/_grid.scss [hunk 1]
```diff
@@ -2,109 +2,110 @@
...
 
-      &:nth-child(3n) {
-        margin-right: 0;
-      }
+    &:nth-child(3n) {
+      margin-right: 0;
     }
   }
+  .usa-width-five-sixths {
+    @include span-columns(5);
```
Comment (maya) [inline src/stylesheets/core/_grid.scss]: I think this should be `@include span-columns(4);`  see recent change here: https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_grid.scss#L36

**Comment by rogeruiz** [comment]
Closing this ticket for now so we can switch over branches easily. We can cherry-pick these changes into a new branch so that we're not committing 309 changes into `18f-pages-staging`. I'm assuming the changes here are just these two commits right? @webinista @maya @msecret 

https://github.com/18F/web-design-standards/pull/972/commits/6a5575982982160f32de1fbc6dc3e9b613db2123

https://github.com/18F/web-design-standards/pull/972/commits/a57c6e3a13f5ab848e1b424f636e25042ddba240

**Pull request closed without merging**